plugins/controlboard/src/ControlBoard.cc

```diff
@@ -24,7 +24,7 @@ namespace gazebo
 GZ_REGISTER_MODEL_PLUGIN(GazeboYarpControlBoard)
 
     GazeboYarpControlBoard::GazeboYarpControlBoard() : m_iWrap(0),
-                                                       m_iVirtAnalogSensorWrap(0)
+                                                       m_iVirtAnalogSensorWrap(nullptr)
```

Comment:
Could you also make relevant changes in the destructor?

Reply:
I think that any 0/nullptr cleanup can be done in a separate PR.

Reply:
> Could you also make relevant changes in the destructor?


Since this is unrelated to the fix suggested in this PR, I shall open another PR for that.